frontend/src/metabase/visualizations/components/TableSimple/TableSimple.styled.tsx

```diff
@@ -71,19 +72,20 @@ export const TableHeaderCellContent = styled.button<{
   isRightAligned: boolean;
 }>`
   display: flex;
-  justify-content: ${props =>
-    props.isRightAligned ? "space-between" : "flex-start"};
+  align-items: center;
+  justify-content: flex-start;
   width: 100%;
+  flex-direction: ${props => (props.isRightAligned ? "row-reverse" : "row")};
 
-  margin-left: ${props => (props.isRightAligned ? "auto" : "unset")};
+  //margin-left: ${props => (props.isRightAligned ? "auto" : "unset")};
```

Comment:
Should we just remove this line?

Reply:
Yes, I forgot it, thanks for pointing out